cli/command/container/create.go

```diff
@@ -118,10 +117,11 @@ type cidFile struct {
 func (cid *cidFile) Close() error {
 	cid.file.Close()
 
-	if !cid.written {
-		if err := os.Remove(cid.path); err != nil {
-			return fmt.Errorf("failed to remove the CID file '%s': %s \n", cid.path, err)
-		}
+	if cid.written {
+		return nil
+	}
+	if err := os.Remove(cid.path); err != nil {
+		return fmt.Errorf("failed to remove the CID file '%s': %s \n", cid.path, err)
```

Comment:
Yes, I actually considered that, but wasn't sure if it would change the format; I'll dig into that at some point :+1: